util/Seccomp.c

```diff
@@ -38,7 +38,7 @@
 static void catchViolation(int sig, siginfo_t* si, void* threadContext)
 {
     printf("Attempted banned syscall number [%d] see doc/Seccomp.md for more information\n",
-           si->si_value.sival_int);
+           si->si_syscall);
```

Comment:
Good idea, si_syscall was added in 3.5 and so was SECCOMP_BPF so it's safe to use in this file.